src/main/java/org/opensearch/neuralsearch/util/HybridQueryUtil.java

```diff
@@ -21,6 +21,8 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class HybridQueryUtil {
 
+    public static final float boost_factor = 1f;
```

Comment:
lets move this to HybridCollectorManagerClass, as it is only getting used there.